kong/error_test.go

```diff
@@ -49,3 +51,36 @@ func TestAPIError_Code(T *testing.T) {
 	require.True(ok)
 	assert.True(kongErr.Code() == 404)
 }
+
+func TestIsForbiddenErrE2E(T *testing.T) {
+	assert := assert.New(T)
+
+	// Create the http client with a custom transport
+	// always returning a 403
+	forbiddenTransport := &forbiddenTransport{}
+	httpClient := &http.Client{
+		Transport: forbiddenTransport,
+	}
+	// if KONG_ADMIN_TOKEN is set, a different custom transport would be used
+	T.Setenv("KONG_ADMIN_TOKEN", "")
+	adminToken := os.Getenv("KONG_ADMIN_TOKEN")
+	os.Unsetenv("KONG_ADMIN_TOKEN")
+	defer func() {
+		os.Setenv("KONG_ADMIN_TOKEN", adminToken)
+	}()
+
+	client, err := NewTestClient(nil, httpClient)
+	assert.NoError(err)
+	assert.NotNil(client)
```

Comment:
Can we use `require` instead of `assert` here? This will stop the test execution when these statements fail.

```suggestion
	require.NoError(T, err)
	require.NotNil(T, client)
```